pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -94,27 +94,28 @@ inline bool rprIsErrorCheck(const TfCallContext &context, const rpr_status statu
 
 #define RPR_ERROR_CHECK(STATUS, MESSAGE_ON_FAIL) rprIsErrorCheck(TF_CALL_CONTEXT, STATUS, MESSAGE_ON_FAIL)
 
-std::string GetRprTmpDir()
-{
+const char* HdRprApi::GetTmpDir() {
 #ifdef WIN32
-
 	char appDataPath[MAX_PATH];
 	// Get path for each computer, non-user specific and non-roaming data.
 	if (SUCCEEDED(SHGetFolderPath(NULL, CSIDL_COMMON_APPDATA, NULL, 0, appDataPath)))
 	{
-		char buff[1024] = {};
-		sprintf(buff, "%s/UsdRpr/", appDataPath);
-		return std::string(buff);
+		static char path[MAX_PATH];
+		snprintf(path, sizeof(path), "%s\\UsdRpr\\", appDataPath);
```

Comment:
Should this folder be called hdRPR or UsdRpr?  I'm slightly leaning towards hdRPR

Reply:
But please give your thoughts.

Reply:
hdRPR is closer to reality because this is actually Hydra plugin